README.md

```diff
@@ -13,14 +15,31 @@ For more information about contributing to the Kubernetes documentation, see:
 * [Using Page Templates](http://kubernetes.io/docs/contribute/style/page-templates/)
 * [Documentation Style Guide](http://kubernetes.io/docs/contribute/style/style-guide/)
 
+## Running the site locally using Hugo
+
+The Kubernetes documentation is built using the [Hugo](https://gohugo.io) static site generator. See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions.
+
+> The site currently requires Hugo version **0.46**.
```

Comment:
As I recall, I installed 0.46 first and the build still failed. I then installed the extended version, checked the Hugo docs which I thought indicated that the `resources` function is available only in the extension. I can re-check if needed.

Reply:
Here is what I found, locally:
- a version previous to 0.46 (ex. 0.42.1) emits a stack trace of errors about the resources method not found
- version 0.46 emits an info message that the resources method is not available in this version
- version 0.46 with extensions works w/o error

Reply:
@kbhawkey That sounds consistent with what I would expect. I think the current recommendation should be fine, though I'll add a note about the `INFO`-level log in case users who have the non-extended version are confused.

Reply:
@lucperkins, I am unable to build with Hugo 0.46. The extended version works, though. Are the two generated files (.content, .json), from a successful build, necessary?

Here is the output using 0.46:
```$ hugo server
Building sites … ERROR 2018/08/16 10:38:23 error: failed to transform resource: TOCSS: failed to transform "sass/styles.sass" (text/x-sass): this feature is not available in your current Hugo version
Total in 19371 ms
Error: Error building site: logged 1 error(s)```

Reply:
Based on outcome of this, `/docs/contribute/intermediate/` will need an update for the section about running locally.